dependencies/src/main/resources/linkage-checker-exclusion-default.xml

```diff
@@ -0,0 +1,55 @@
+<LinkageCheckerFilter>
+  <!-- reactor-core's Traces catches Throwable to detect classes available in
+    Java 9+. As Linkage Checker targets Java 8, it ignores these errors to avoid
+    false positives.
+    https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/816 -->
+  <LinkageError>
+    <Source>
+      <Package name="reactor.core.publisher.Traces" />
+    </Source>
+  </LinkageError>
+
+  <!-- GraalVM-related libraries depend on Java Compiler Interface (JVMCI) that
+    only exists in special JDK. These missing classes are false positives, because
+    the code is only invoked when running in a GraalVM.
+    https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/929 -->
+  <LinkageError>
+    <Target>
+      <Package name="jdk.vm.ci" />
+    </Target>
+    <Source>
+      <Package name="com.oracle.svm" />
+    </Source>
+  </LinkageError>
+  <LinkageError>
+    <Target>
+      <Package name="jdk.vm.ci" />
+    </Target>
+    <Source>
+      <Package name="com.oracle.graal" />
+    </Source>
+  </LinkageError>
+  <LinkageError>
+    <Target>
+      <Package name="jdk.vm.ci" />
+    </Target>
+    <Source>
+      <Package name="org.graalvm" />
+    </Source>
+  </LinkageError>
+
+  <!-- Mockito's MockMethodDispatcher uses special class loader to load
+    MockMethodDispatcher.raw. The class file with "raw" suffix is in mockito-core JAR file.
+    As long as developers use Mockito through a normal usage, there should not be
```

Comment:
delete the sentence beginning "As long as"

Reply:
Deleted.